MAINTAINERS

```diff
@@ -0,0 +1,16 @@
+Avocado-vt maintainers
+======================
+
+The intention of this file is not to establish who owns what portions of the
+code base, but to provide a set of names that developers can consult when they
+have a question about a particular subset and also to provide a set of names
+you can look for on github for any pull requests you might want to get approved.
+
+In general, if you have a question, you should send an email to the Virt Test
+development mailing list and not any specific individual privately.
+
+L: Avocado-devel <avocado-devel@redhat.com>
+M: Yanbing Du <ydu@redhat.com>
+M: Jiangang Wei <weijg.fnst@cn.fujitsu.com>
+M: Guannan Wayne Sun <gsun@redhat.com>
+M: Hao Liu <hliu@redhat.com>
```

Comment:
@xutian I can only talk for myself, but I can't focus on avocado-vt fully. I'll still be active, but not as a primary contact and I think it's the same with @clebergnu. So far only guys from libvirt answered, so I put them there and remove the sections. We can always bring the sections back if more people volunteer to be the maintainers of a specific part.

@waynesun09 I could put @xutian instead of you, but he did not volunteered. So @xutian do you want to be included here instead of @waynesun09?